public/apps/configuration/utils/__snapshots__/panel-with-header.test.tsx.snap

```diff
@@ -0,0 +1,40 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Panel with header renders 1`] = `
+<EuiPanel>
+  <EuiTitle
+    size="s"
+  >
+    <h3>
+      Header
+      <i>
+         - optional
+      </i>
+      <span
+        className="panel-header-count"
+      >
+         (
+        0
+        )
```

Comment:
This style looks a bit strange. Can you fix this?

Reply:
Actually it is correct style. I checked again. We need space before first parenthesis. It looks good in UI.